packages/core/devTools/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "main": "dist/xstate.cjs.js",
+  "module": "dist/xstate.esm.js",
+  "preconstruct": {
+    "source": "../src/devTools"
```

Comment:
nit: maybe `dev-tools`? somehow this would look better for me as part of the public export, but I also don't care that much, so please chose whatever you like

Reply:
Changed to `dev`